pkg/server/updateResults.go

```diff
@@ -133,7 +133,7 @@ func postUpdateResult(updateResults []UpdateResults) error {
 func getTimeWindows(days int, windowSize int) []time.Time {
 	var windows []time.Time
 	currentTime := time.Now()                  // To Do : we should get the time from the input
-	startTime := currentTime.AddDate(0, 0, -5) // Go back to five days
+	startTime := currentTime.AddDate(0, 0, -1) // Go back to one days
```

Comment:
I remember not getting enough data with this set to 1, that's why it was increased to 5 - but maybe there was fix in recent kruize.